src/library/pagination/pagination.component.js

```diff
@@ -0,0 +1,170 @@
+// Import Template
+import template from './pagination.html';
+
+class controller {
+  constructor($log, $scope) {
+    Object.assign(this, {
+      $log,
+      $scope
+    });
+
+    this.currentPage = 0; // binding
+    this.disabled = false; // binding
+    this.itemsPerPage = 0; // binding
+    this.size = ''; // binding
+    this.totalItems = 0; // binding
+    this.visiblePageBuffer = 0; // binding
```

Comment:
It's a habit, really.  I will yank them out.

It's mainly for readability and maintenance purposes.  I always like that a class can stand on its own without needing to reference any documentation, or in this case, the component definition.  Since JavaScript isn't typed, these are simply hints to show the developer what values will end up in those properties.  I've often thought about using `Object.seal` in the constructor as well, to limit side effects, but haven't gone that far.  Sometimes it's nice to be able to set a property on an object you don't know... that's sort of the joy of JavaScript. ;)